client/src/components/layouts/variables.css

```
@@ -42,6 +42,7 @@
   --highlight-color: var(--blue-light);
   --highlight-background: var(--blue-dark);
   --selection-color: var(--blue-light-translucent);
+  --focus-outline-color: var(--blue-light);
```

```suggestion
  --focus-outline-color: var(--blue-mid);
```

The outline color for both light/dark needs to stay with blue-mid. This color of blue meets color contrast requirements for both themes at the same time. blue-light has issues in the dark theme and blue-dark has issues in the light theme.

---

Okay but because they are the same, could I simply define it at the root?